CHANGELOG.md

```diff
@@ -2,6 +2,19 @@
 All notable changes to this project will be documented in this file. This
 project adheres to [Semantic Versioning](http://semver.org/).
 
+## Current
+### Changed
+- Only return `URN` instances for valid `URN`s at creation. Raise `URN::InvalidURNError` otherwise.
+- `#valid?` is deprecated and always return true
```

Comment:
As 2.0 already has major, breaking API changes, could we just remove `valid?` altogether?